resources/js/Components/twc-ui/description-list.tsx

```
@@ -0,0 +1,18 @@
+import type React from 'react'
+import { cn } from '@/Lib/utils'
+
+const DescriptionList = ({ className, ref, ...props }: React.ComponentProps<'dl'>) => {
+  return <dl ref={ref} className={cn('text-sm space-y-0', className)} {...props} />
```

Biome (2.3.13)

[error] These CSS classes should be sorted.

This rule is still being actively worked on, so it may be missing features or have rough edges. Visit https://github.com/biomejs/biome/issues/1274 for more information or to report possible bugs.
This rule belongs to the nursery group, which means it is not yet stable and may change in the future. Visit https://biomejs.dev/linter/#nursery for more information.
Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)